src/Module/Attach.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @file src/Module/Attach.php
+ */
+
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\L10n;
+use Friendica\Core\System;
+use Friendica\Core\Logger;
+use Friendica\Model\Attach as MAttach;
+
+/**
+ * @brief Attach Module
+ */
+class Attach extends BaseModule
+{
+	/**
+	 * @brief Module initializer
+	 *
+	 * Fetch an attached file given the id
+	 */
+	public static function init()
```

Comment:
Given the nature of this method outputting content directly and exiting, this should rather be `rawContent`to avoid confusion.